bin/kyuubi

```diff
@@ -20,7 +20,7 @@
 CLASS="org.apache.kyuubi.server.KyuubiServer"
 
 function usage {
-  echo "Usage: ./bin/kyuubi (start|stop|status)"
+  echo "Usage: ./bin/kyuubi (start|run|stop|status)"
```

Comment:
Adding run here make the usage not clear anymore，we need describe each option here

Reply:
updated